src/sparseml/pytorch/image_classification/export.py

```diff
@@ -366,9 +381,18 @@ def main(
     if labels_to_class_mapping is None:
         labels_to_class_mapping = helpers.label_to_class_mapping_from_dataset(dataset)
 
-    if recipe is not None:
-        ScheduledModifierManager.from_yaml(recipe).apply_structure(model)
-        load_model(checkpoint_path, model, strict=True)
+    # training recipe must be a part of checkpoint
+    load_model(checkpoint_path, model, strict=True)
+
+    if one_shot:
+        if not recipe:
```

Comment:
Do we need to cover the case where `recipe==True` and `one_shot==False`?

Reply:
Good catch! Updating.